src/Pages/Status/StatusPage.js

```diff
@@ -0,0 +1,172 @@
+import React, { PureComponent } from 'react';
+import AsyncImage from '../Utils/AsyncImage';
+import Container from '../Utils/Container';
+import Metatags from '../Utils/Metatags';
+import Facepalm from './images/facepalm.gif';
+import ServiceStatus from './components/ServiceStatus';
+import StatusPageMetatags from './StatusPageMetatags';
+import Success from './images/success.gif';
+
+const STATUS_RESPONSE_TYPE = {
+  UNKNOWN: 0,
+  SUCCESS: 1,
+  ERROR: 2
+};
+
+class StatusPage extends PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = this.defaultState;
+  }
+
+  defaultState = {
+    calendarApi: STATUS_RESPONSE_TYPE.UNKNOWN,
+    calendarBot: STATUS_RESPONSE_TYPE.UNKNOWN,
+    eventbriteApi: STATUS_RESPONSE_TYPE.UNKNOWN,
+    meetupApi: STATUS_RESPONSE_TYPE.UNKNOWN,
+    spreadsheetApi: STATUS_RESPONSE_TYPE.UNKNOWN
+  };
+
+  checkStatus = () => {
+    this.setState(this.defaultState);
+
+    this.getServiceStatus('https://spreadsheet-api.now.sh/').then((status) => {
```

Comment:
Consider having a fn creator where you pass a servicer url and a state prop so this way you are gonna be able to express all the variations in one fn